.github/workflows/osv-scanner-reusable-pr.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: OSV-Scanner PR scanning
+
+on:
+  workflow_call:
+
+jobs:
+  scan-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      # - name: Get changed files
+      #   id: changed-files
+      #   uses: tj-actions/changed-files@v37
+
+      #     # To compare changes between the current commit and the last pushed remote commit set `since_last_remote_commit: true`. e.g
+      #     # with:
+      #     #   since_last_remote_commit: true 
+
+      # - name: List all changed files
+      #   run: |
+      #     echo ${{ steps.changed-files.outputs.all_changed_files }}
+      #     echo ------------
+      #     for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+      #       echo "$file was changed"
+      #     done
+      - name: "Checkout target branch"
+        run: git checkout $GITHUB_BASE_REF
+      - name: "Run scanner"
+        uses: another-rex/osv-scanner@markdown-output
```

Comment:
Yes we do, but only after this is already merged, otherwise I think it will prevent merging as the action will fail. Maybe a better way is to merge only the code part and have a separate PR for the workflows part.